src/App.jsx

```diff
@@ -14,70 +17,72 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
-
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+export const App = () => {
+  const [selectedGood, setselectedGood] = useState(goods[8]);
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+  const changeSelectedGood = good => () => setselectedGood(good);
```

Comment:
Why did you use closure here?
```suggestion
  const changeSelectedGood = good =>  setselectedGood(good);
```

Reply:
const changeSelectedGood = good =>  setselectedGood(good) - коли ось так переписала, то взагалі нічого не відображає на сторінці...